src/currency-field/CurrencyField.ts

```diff
@@ -140,15 +157,43 @@ export class CurrencyField extends OmniFormElement {
         }
     }
 
-    // Check if the device is a Iphone or Ipad running IOS.
-    _isIOS() {
-        return (
-            ['iPad Simulator', 'iPhone Simulator', 'iPod Simulator', 'iPad', 'iPhone', 'iPod'].includes(navigator.platform) ||
-            // iPad on iOS 13 detection
-            navigator.userAgent.includes('Mac')
+    // Dispatch a custom change event required as we manipulate and format the value of the input.
+    _dispatchCustomEvent(amount: number) {
```

Comment:
Shouldnt this function name be `dispatchChange` so as to indicate which event it actually dispatches?